api/v1beta1/openstackbaremetalset.go

```diff
@@ -90,10 +90,10 @@ func VerifyBaremetalSetScaleUp(
 	l logr.Logger,
 	instance *OpenStackBaremetalSet,
 	allBmhs *metal3v1.BareMetalHostList,
-	existingBmhs *metal3v1.BareMetalHostList) ([]string, error) {
+	existingBmhs *metal3v1.BareMetalHostList) ([]metal3v1.BareMetalHost, error) {
 	// How many new BaremetalHost allocations do we need (if any)?
```

Comment:
This function uses `instance.Spec.BmhLabelSelector` when checking for scale-up. I guess we also have to match the label in the instancespec if there is one?

Reply:
Fair point.  We should fail early in the webhook for the new individual selector label just like we do for the existing group selector label.